docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -104,7 +104,8 @@
     }
 
     figcaption {
-      @include hb-caption($italic: true);
+      @include hb-caption-credit;
+      padding: hb-calculate-rems(16px) hb-calculate-rems(7px) hb-calculate-rems(8px) 15%; // Using a percentage to the left to control the width of the captions
```

Comment:
Ack! 🙈  
I should have looked closer at the padding. It has the same values here and in the `_caption-credit.scss` file. We could probably move this into the `hb-caption-credit` mixin as well.

And I noticed that there's a media query that changes the padding for larger screen sizes. Let's add that as well. The screenshot below is a side-by-side of a WYSIWYG caption on the left and a postcard image caption on the right. You can see that the padding is different. For consistency sake, let's make both the same.

![padding media query](https://user-images.githubusercontent.com/12678977/107062836-b4799b80-67a7-11eb-870c-731b7d9badfa.png)

Reply:
Here's the postcards page I was using for reference: http://sparkbox-sandbox.suhumsci.loc/qa/qa-postcards. You may need to add a caption to a postcard image.